Bottom: 9bcfa88b39a3d715be1df56c513d6bd08dad6917
Top:    84b1935754b1ad2160e3c37da5c1001afd7f9cee
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-10 20:43:14 -0500

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
